### DISCUSSION TITLE
What are the axes for?

### DISCUSSION THREAD

**Discussion by BennyDS** [Kegetys/simagic-arduino-poc#2] [Q&A]
Hi 
First off, thank you so much for this. Definitly going to try this to add some buttons to my karting steering wheel.
I went through the code, I think I understand most of it to make it work for myself.
I dont understand what the axes are for. Are these analogue handclutches? If I dont use them, do you recommend taking them out of the code or just leave as is?
I own a alpha mini base, but i dont have a Simagic steering wheel. So I dont know what all the bells and whistles are on those wheels.

Thank you.

**Comment by Anguiano76** [comment]
Hello.

Did you manage to connect a button? I get it to connect and appear in SimPro Manager, but I have no idea how to connect the buttons.

Greetings and thanks
